tests/integration/components/bubble-chart-test.js

```diff
@@ -0,0 +1,75 @@
+import Service from '@ember/service';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import crossfilter from 'crossfilter';
+import wait from 'ember-test-helpers/wait';
+import { later } from '@ember/runloop';
+
+const getTestParameters = function () {
+    const rawData
+    = [{ 'fruit': 'blueberries', 'radius': 40, 'description': 'delicious' },
+        { 'fruit': 'oranges', 'radius': 30, 'description': 'ew' },
+        { 'fruit': 'apples', 'radius': 10, 'description': 'tasty' },
+        { 'fruit': 'strawberries', 'radius': 50, 'description': 'no good' }];
```

Comment:
This is a little bit of a nitpick but maybe for consistency with the rest of our code we can do

```
const rawData = [
    { 'fruit': 'blueberries', 'radius': 40, 'description': 'delicious' },
    { 'fruit': 'oranges', 'radius': 30, 'description': 'ew' },
    { 'fruit': 'apples', 'radius': 10, 'description': 'tasty' },
    { 'fruit': 'strawberries', 'radius': 50, 'description': 'no good' }
];
```

Reply:
👍